doc/tutorials.rst

```diff
@@ -90,6 +90,11 @@ algorithms using PennyLane and near-term quantum hardware.
     :figure: ../examples/figures/barren_plateaus/surface.png
     :description: :ref:`barren_plateaus`
 
+.. customgalleryitem::
+    :tooltip: Doubly stochastic gradient descent
+    :figure: ../examples/figures/doubly_stochastic/single_shot.png
```

Comment:
Can we remove the legend and axis labels in the thumbnail? You can't read them anyway

Reply:
Done:
![Screenshot_20191007_172341](https://user-images.githubusercontent.com/2959003/66349600-3e50f480-e927-11e9-91c8-2a080fde867b.png)

Reply:
Nice

Reply:
I see it the other way: this plot is in fact a very clear indication that the stochastic estimators can have high variance, but still (on average) find the minimum